### PR TITLE
update sheets + annotate RuanMei sheets

### PR DESCRIPTION
**Pull request by frzyc**
## Describe your changes

Update some sheet UI stuff, fixing some issues in sheets, adding TODOs to translation and gaps in sheet interactions

## Issue or discord link

- 

## Testing/validation

image

## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)

- [ ] I have commented my code in hard-to understand areas.
- [ ] I have made corresponding changes to README or wiki.
- [ ] For front-end changes, I have updated the corresponding English translations.
- [ ] I have run `yarn run mini-ci` locally to validate format and lint.
- [ ] If I have added a new library or app, I have updated the deployment scripts to ignore changes as needed

### CODE REVIEW

File: libs/sr/formula/src/data/char/sheets/RuanMei.ts [hunk 1]
```diff
@@ -41,6 +41,8 @@ const sheet = register(
 
   // Formulas
   ...dmg('basicDmg', baseTag, 'atk', dm.basic.dmg, 'basic'),
+  // TODO: should be Ice Break DMG
+  // TODO: should be hidden behind ult conditional
```
Comment (nguyentvan7) [inline libs/sr/formula/src/data/char/sheets/RuanMei.ts:45]: we don't hide damage instances behind conditionals. Tho i guess we could if we have conditional-per-target, but seems clunky

File: libs/sr/formula/src/data/char/sheets/RuanMei.ts [hunk 2]
```diff
@@ -51,6 +53,7 @@ const sheet = register(
     { ...baseTag, damageType1: 'break' },
     subscript(char.talent, dm.talent.breakDmg)
   ),
+  // TODO: shouldn't this be incorporated to the talentBreakDmg above?
```
Comment (nguyentvan7) [inline libs/sr/formula/src/data/char/sheets/RuanMei.ts:56]: why?

File: libs/sr/formula/src/data/char/sheets/RuanMei.ts [hunk 3]
```diff
@@ -41,6 +41,8 @@ const sheet = register(
 
   // Formulas
   ...dmg('basicDmg', baseTag, 'atk', dm.basic.dmg, 'basic'),
+  // TODO: should be Ice Break DMG
```
Comment (nguyentvan7) [inline libs/sr/formula/src/data/char/sheets/RuanMei.ts:44]: It is

Comment (frzyc) [reply libs/sr/formula/src/data/char/sheets/RuanMei.ts:44]: oops, meant as a TODO to show the "ice variant" on the UI side. this TODO is in the wrong place

File: libs/sr/formula/src/data/char/sheets/RuanMei.ts (hunk 2 above)
Comment (frzyc) [reply libs/sr/formula/src/data/char/sheets/RuanMei.ts:56]: is e6 a separate instance of dmg from the talent hit? i read it as an increase in multiplier (same instance of dmg)

Comment (nguyentvan7) [reply libs/sr/formula/src/data/char/sheets/RuanMei.ts:56]: True it should be added

**Review by nguyentvan7 — APPROVED** [review]

**Pull request merged**